ecs-agent/netlib/model/ecscni/vpctunnel_config.go

```diff
@@ -68,12 +71,18 @@ func (c *VPCTunnelConfig) PluginName() string {
 
 // SetV2NDstPortAndDeviceName assigns a destination port to the task ENI and assigns
 // it a device name with the pattern gnv<vni><dst port>.
-func SetV2NDstPortAndDeviceName(taskENI *networkinterface.NetworkInterface, dstPort uint16) {
-	vni := taskENI.TunnelProperties.ID
-	taskENI.TunnelProperties.DestinationPort = dstPort
```

Comment:
Sorry I forgot - is the previous code in "unimplemented" state? Asked because this is a behavior change if we already use it.

Reply:
No this function was not being used until now.